components/gitpod-protocol/src/gitpod-service.ts

```diff
@@ -344,6 +344,8 @@ export namespace WorkspaceTimeoutDuration {
 export const WORKSPACE_TIMEOUT_DEFAULT_SHORT: WorkspaceTimeoutDuration = "30m";
 export const WORKSPACE_TIMEOUT_DEFAULT_LONG: WorkspaceTimeoutDuration = "60m";
 export const WORKSPACE_TIMEOUT_EXTENDED: WorkspaceTimeoutDuration = "180m";
+export const WORKSPACE_LIFETIME_SHORT: WorkspaceTimeoutDuration = "8h";
+export const WORKSPACE_LIFETIME_LONG: WorkspaceTimeoutDuration = "36h";
```

Comment:
Should these really be hard-coded? It seems these values are SaaS specific, but not necessarily dedicated specific.